tmt/steps/report/reportportal.py

```diff
@@ -1,30 +1,27 @@
 import dataclasses
-import io
 import os
-import types
-import zipfile
+from time import time
 from typing import Optional
 
+import requests
+
 import tmt.steps.report
 import tmt.utils
 
-from . import junit
-
-junit_xml: Optional[types.ModuleType] = None
 
+def timestamp() -> str:
+    return str(int(time() * 1000))
 
-def import_junit_xml() -> None:
-    """
-    Import junit_xml module only when needed
 
-    Until we have a separate package for each plugin.
-    """
-    global junit_xml
-    try:
-        import junit_xml
-    except ImportError:
-        raise tmt.utils.ReportError(
-            "Missing 'junit-xml', fixable by 'pip install tmt[report-junit]'.")
+def get_status(result: tmt.result.ResultOutcome) -> str:
+    if result == tmt.result.ResultOutcome.PASS:
+        return "PASSED"
+    elif result == tmt.result.ResultOutcome.FAIL \
+            or result == tmt.result.ResultOutcome.ERROR \
+            or result == tmt.result.ResultOutcome.WARN:
```

Comment:
I suggest to use `in` operator: `result in tmt.result.ResultOutcome.FAIL, tmt.result.ResultOutcome.ERROR, tmt.result.ResultOutcome.WARN`

Also to have that line shorter, try something like `from tmt.result import ResultOutcome` if possible.

Reply:
Thank you, already updated